doc/nimc.rst

```diff
@@ -228,6 +228,42 @@ configuration file should contain something like::
   arm.linux.gcc.exe = "arm-linux-gcc"
   arm.linux.gcc.linkerexe = "arm-linux-gcc"
 
+Cross compilation for Nintendo Switch
+=====================================
+
+Simply add --os:nintendoswitch
+to your usual ``nim c`` or ``nim cpp`` command. DevkitPro setup must be the same as
+what is the default with their new installer
+[here for Mac/Linux](https://github.com/devkitPro/pacman/releases) or
```

Comment:
That's not how RST links look like.

Reply:
Oops, copy pasted from MD file. I'll update to be in line with rst.